### PR TITLE
Release/0.4.0

### PR DESCRIPTION
**Pull request by adatzer**

### CODE REVIEW

File: template.tpl [hunk 1]
```diff
@@ -186,252 +186,604 @@ ___TEMPLATE_PARAMETERS___
...
+                ]
               }
-            ],
-            "enablingConditions": [
+            ]
+          },
+          {
+            "type": "GROUP",
+            "name": "globalEntitiesGroup",
+            "displayName": "Global context entities",
```
Comment (paulboocock) [inline template.tpl]: I don't think its clear from the Tag UI what a custom entity and a global entity is. The difference in the UI is very subtle (no event name) and I had to read the code before I noticed what the difference was! I think adding in the help text somewhere that helps users understand the difference would be good. I'm not sure help text renders on groups, but maybe you could add a label into the group?

> Custom Entities are attached where the event_name matches the incoming event, whereas global entities will be applied to all events

**Review by paulboocock — APPROVED** [review]
This tag is moderately insane now but very powerful! One small comment but other than that, looks good to me!

File: template.tpl (hunk 1 above)
Comment (adatzer) [reply template.tpl]: Thank you for reviewing @paulboocock ! Indeed help texts on groups do not seem to render, so added it in a label under a "Context entities settings" group.
This is what it looks like

context_entities_settings

Labels render on tag summary as well (no option to disable). Is this ok?

sp_tag_summary

Comment (paulboocock) [reply template.tpl]: Can we try and add the label to the help text of the table, or something lower down the nesting so it doesn't show at this level?

Comment (adatzer) [reply template.tpl]: Labels insist on showing up at summary level, so just splitted the help text for custom and global and added it to the respective tables' help text:

select-area_20221122132929

**Review by paulboocock — APPROVED** [review]
Awesome! Love the new tool tips!

**Pull request merged**